weather_dl/download_pipeline/pipeline_test.py

```diff
@@ -632,5 +635,19 @@ def test_target_name_date_and_target_directory_additional_partitions_in_path(sel
         self.assertEqual(target_name, "somewhere/expver-1/2017/01/15-pressure-500.nc")
 
 
+class ConfigTest(unittest.TestCase):
+
+    def setUp(self):
+        self._data_dir = f'{next(iter(weather_dl.__path__))}/../configs'
+
+    def test_process_config_files(self):
+        for filename in os.listdir(self._data_dir):
+            config = os.path.join(self._data_dir, filename)
+            try:
+                run(["weather-dl", config, "--dry-run"])
+            except Exception as e:
+                assert False, ("\n" + type(e).__name__ + " " + str(e) + "\nIn " + filename)
```

Comment:
This assertion statement is a bit messy. I'd suggest you consider applying one of two alternatives (the former is more conventional): 

- Instead of `assert False`, consider using `self.fail()`. Then, use an `f` string to template the failure statement
- ```
   raise AssertionError(f'Config {filename!r} is incorrect') from e
   ```